src/components/EditorSideBar/index.tsx

```diff
@@ -0,0 +1,17 @@
+import EditorSideBarItem from "./EditorSideBarItem";
+import sideBarItemList from "./const";
+import { Box, Flex } from "@chakra-ui/react";
+
+const EditorSideBar = () => {
+  return (
+    <Box bg="background.primary" w="98px" h="calc(100vh)" position="fixed" top="71px" left="0">
```

Comment:
Also top should be 72px. There is rarely use of odd numbers on UI elements.(unless the element is calculated by itself)